components/homme/src/share/derivative_mod_base.F90

```diff
@@ -11,7 +11,7 @@ module derivative_mod_base
   use parallel_mod,   only : abortmp
   use element_mod,    only : element_t
   use control_mod,    only : hypervis_scaling, hypervis_power
-  use physical_constants, only : rrearth
+  use physical_constants, only : scale_factor_inv, laplacian_rigid_factor
```

Comment:
Just curious -- why does laplacian have its rigid factor but not other derivatives? what is rigid factor?

Reply:
nm, see it below